docs/data/data-grid/performance/performance.md

```diff
@@ -2,43 +2,51 @@
 
```

Comment:
Should we explain a bit the reactive behavior of `useGridSelector` thanks to the Store possibly with a small demo so that users start using it more often in customizations?

Reply:
I've explained a bit more this one in the docs for the reactive part.